cluster_experiments/washover.py

```diff
@@ -97,46 +97,66 @@ def washover(
         cluster_cols: List[str],
         original_time_col: Optional[str] = None,
     ) -> pd.DataFrame:
-        """No washover - returns the same dataframe as input.
+        """Constant washover - we drop all rows in the washover period when
+        there is a switch where the treatment is different.
 
         Args:
             df (pd.DataFrame): Input dataframe.
             truncated_time_col (str): Name of the truncated time column.
             treatment_col (str): Name of the treatment column.
-            cluster_cols (List[str]): List of clusters of experiment.
-            original_time_col (Optional[str], optional): Name of the original time column.
+            cluster_cols (List[str]): List of clusters of experiment. The list must include the 'truncated_time_col' as well.
+            original_time_col (str): Name of the original time column.
```

Comment:
also revert this line:
original_time_col (Optional[str], optional): Name of the original time column.